client/src/components/NextPageButton/index.js

```diff
@@ -17,7 +18,7 @@ const NextPageButton = () => {
   }
 
   return (
-    <div className='PageButton' onClick={onClick}>
+    <div className={c.ageButton} onClick={onClick}>
```

Comment:
Is this a typo?.. Should it be "c.PageButton" instead of "c.ageButton" ?

Reply:
oh yeah it is, thanks for catching that!